### PR TITLE
Github feature request template: Fix label & improve paragraph order

### PR DESCRIPTION
**Pull request by hoffie**
**Short description of changes**

- Github: Fix label in feature request template
  This template used a label which did not exist (anymore).

- Github: Improve paragraph order in feature request template
  It makes most sense to ask users about context first, about different solutions second, about the preferred strategy third and about the agreement last.

CHANGELOG: SKIP

**Context: Fixes an issue?**

I always struggle to fill out the current feature request template. It seems upside down to me. Therefore, I'm proposing to change it. :)

**Does this change need documentation? What needs to be documented and how?**

/

**Status of this Pull Request**

**What is missing until this pull request can be merged?**

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I tested my code and it does what I want
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -2,17 +2,20 @@
...
+-->
 
-**Describe the solution you'd like**
-<!-- A clear and concise description of what you want to happen. -->
+<!-- Describe the solution you'd like -->
 
-**Describe alternatives that have been considered**
-<!-- If some alternatives have been discussed, please briefly list those together with a reason(s) why they were decided against. This prevents a repeat of the previous discussions. Reference any relevant discussion threads if possible, and consider tagging the main participants. -->
+**Has this feature been discussed and generally agreed?**
+<!-- Don't worry about requesting something that hasn't been discussed - we'll just move it to Discussion if we think it's unclear. But ideally, issues should have an agreed specification such that anyone could action it without much help. -->
```
Comment (ann0see) [inline .github/ISSUE_TEMPLATE/feature_request.md:21]:
```suggestion
<!-- Don't worry about requesting something that hasn't been discussed - we'll just move it to Discussion if we think it's unclear. But ideally, issues should have an agreed, clear specification such that anyone could action it without much help. -->
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -2,17 +2,20 @@
...
 about: Agreed specification for a feature or enhancement
 title: ''
-labels: 'feature'
+labels: 'feature request'
 assignees: ''
 ---
 
-<!-- Thanks for specifying a new feature! -->
+**Provide context**
+<!-- What's the status quo and how is this non-optimal?** -->
```
Comment (ann0see) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: I don't think this is clear enough (at least for me "giving context" could mean anything). Can you formulate the headline as question too?

Maybe: What is the current behaviour and why should it be changed?
Comment: Please give a precise description of the status quo and how this is not optimal.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -2,17 +2,20 @@
...
 
-**Has this feature been discussed and generally agreed?**
-<!-- Don't worry about requesting something that hasn't been discussed - we'll just move it to Discussion if we think it's unclear. But ideally, issues should have an agreed specification such that anyone could action it without much help. -->
+**Describe possible approaches**
```
Comment (ann0see) [inline .github/ISSUE_TEMPLATE/feature_request.md:12]: I think the user should also be lead to think about KISS. Can this be accieved by another way? Another software? Why should this be supported by Jamulus?
Comment: Please note the contribution guidelines (link here), especially that new features should be added with caution to keep Jamulus simple.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -2,17 +2,20 @@
...
+<!--
+If multiple approaches are possible, list them here with pros and cons.
```
Comment (ann0see) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]:
```suggestion
Think about other approaches and list them here with pros and cons if possible.
```

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (hoffie) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Updated (more or less verbatim).

**Review by ann0see — APPROVED** [review]
Great. Waiting on other review.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (hoffie) [reply .github/ISSUE_TEMPLATE/feature_request.md:12]: Updated. Can you re-check?

I agree with all of your points, but I think it has downsides to have rather long explanations, too. People may either ignore it or walk away with some "too complicated" feeling.

But as I mainly care about the order, I've added your suggestions.

Comment (ann0see) [reply .github/ISSUE_TEMPLATE/feature_request.md:12]: Yes, thought of that too. Let me think about something else...

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -2,17 +2,22 @@
...
+<!-- Please give a precise description of the status quo and why this is not optimal. -->
 
-**Has this feature been discussed and generally agreed?**
-<!-- Don't worry about requesting something that hasn't been discussed - we'll just move it to Discussion if we think it's unclear. But ideally, issues should have an agreed specification such that anyone could action it without much help. -->
+**Describe possible approaches**
+<!--
+List possible approaches here with pros and cons if possible.
+Consider both Jamulus-internal changes as well as external interfaces or solutions.
+Reference any relevant discussion threads if possible, and consider tagging the main participants.
+Please note the contribution guidelines (https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md), especially that new features should be added with caution to keep Jamulus simple.
```
Comment (ann0see) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]:
```suggestion
List possible approaches with pros and cons, reference any relevant discussion threads if possible, and consider tagging the main participants.
Please note the contribution guidelines (https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md), especially the part about "keeping it simple and stupid".
```

**Comment by gilgongo** [comment]
Looks good - and as it says we can always move to Discussion if needed. Waiting for the day when GitHub see sense and allow you to apply permissions to Issues, but things have been OK and most people seems to go to the Discussions first now.

**Review by gilgongo — APPROVED** [review]

**Comment by ann0see** [comment]
Why did code Ql fail again?!

@hoffie I hope re-running this makes the CI green.

**Review by pljones — APPROVED** [review]

**Pull request merged**